app/api/tasks.js

```diff
@@ -9,7 +9,9 @@ const byEntitiesUris = name => uris => {
 export default {
   byIds (ids) { return action('by-ids', { ids }) },
   byScore (limit, offset) { return action('by-score', { limit, offset }) },
+  byEntitiesType (params) { return action('by-entities-type', params) },
   bySuspectUris: byEntitiesUris('suspect'),
   bySuggestionUris: byEntitiesUris('suggestion'),
+  deduplicateWork: action('deduplicate-work'),
```

Comment:
update required after https://github.com/inventaire/inventaire/commit/b2a18f9
```suggestion
  deduplicateWorks: action('deduplicate-works'),
```

Reply:
reviews are precious :)